docs/modules/databend.md

```diff
@@ -0,0 +1,55 @@
+# Databend
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for Databend.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the Databend module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/databend
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a Databend container](../../modules/databend/examples_test.go) inside_block:runDatabendContainer
+<!--/codeinclude-->
+
+## Module Reference
+
+### Run function
+
+- Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+!!!info
+    The `RunContainer(ctx, opts...)` function is deprecated and will be removed in the next major release of _Testcontainers for Go_.
+
+The Databend module exposes one entrypoint function to create the Databend container, and this function receives three parameters:
+
+```golang
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*DatabendContainer, error)
+```
+
+- `context.Context`, the Go context.
+- `string`, the Docker image to use.
+- `testcontainers.ContainerCustomizer`, a variadic argument for passing options.
+
+### Container Options
+
+When starting the Databend container, you can pass options in a variadic way to configure it.
+
+#### Image
+
+If you need to set a different Databend Docker image, you can set a valid Docker image as the second argument in the `Run` function.
+E.g. `Run(context.Background(), "datafuselabs/databend:v1.2.615")`.
+
+{% include "../features/common_functional_options.md" %}
+
```

Comment:
@hantmac it would be convenient to document the new functional options for databend here at this point. Please use other modules as reference (e.g. MySQL: https://golang.testcontainers.org/modules/mysql/#set-username-password-and-database-name)

In this PR, the module is exposing two options: https://github.com/testcontainers/testcontainers-go/pull/2779/files#diff-05e51be59eadcbdc57a90dccc140946369a930b7bec7e9e987096ebed8beb9beR127, and two container methods (see https://github.com/testcontainers/testcontainers-go/pull/2779/files#diff-05e51be59eadcbdc57a90dccc140946369a930b7bec7e9e987096ebed8beb9beR103). Please add the container methods right after L55: https://github.com/testcontainers/testcontainers-go/pull/2779/files#diff-b3c18d18db5bfc68feda99c0eb84554cb77251863f0f2a5237133eced8fbc72eR55

Reply:
BTW the render URL of the docs for this PR is here: https://deploy-preview-2779--testcontainers-go.netlify.app/modules/databend/

Reply:
Thanks for your help! I fixed the dcs.